test/fft_base.py

```diff
@@ -467,7 +471,7 @@ def setUp(self):
         self.in_c2c_rev = [3.0-1.0j,-1.0+3.0j]
         self.out_c2c_rev = [2.0+2.0j,4.0-4.0j]
         # For Time/FrequencySeries, we want to test with a non-trivial epoch
-        self.epoch = LTG(3,4)
+        self.epoch = 3 + 4 * 1e-9
```

Comment:
This is the only change in the file that is actually related to the epoch change - the rest is general cleanup